tests/helper/charts/rancherbackuprestore.go

```diff
@@ -24,15 +27,30 @@ const (
 	RancherBackupRestoreName          = "rancher-backup"
 	RancherBackupRestoreCRDName       = "rancher-backup-crd"
 	BackupRestoreConfigurationFileKey = "../helper/yamls/inputBackupRestoreConfig.yaml"
+	localStorageClass                 = "../helper/yamls/localStoarageClass.yaml"
```

Comment:
Typo in localStorageClass name

```suggestion
	localStorageClass                 = "../helper/yamls/localStorageClass.yaml"
```